python/tank/log.py

```diff
@@ -315,12 +315,17 @@ def doRollover(self):
                 self._handle_rename_failure("w")
                 return
 
+            # Python 2.6 expects the file to be opened during rollover.
+            if not self.stream and sys.version_info[:2] < (2, 7):
+                self.mode = "a"
+                self.stream = self._open()
+
             # Now, that we are back in the original state we were in,
             # were pretty confident that the rollover will work. However, due to
             # any number of reasons it could still fail. If it does, simply
             # disable rollover and append to the current log.
             try:
-                super(LogManager._SafeRotatingFileHandler, self).doRollover()
+                logging.handlers.RotatingFileHandler.doRollover(self)
```

Comment:
minor: maybe we could import `logging.handlers.RotatingFileHandler` so that we can refer to it just as `RotatingFileHandler`?